apple/internal/stub_support.bzl

```diff
@@ -60,11 +60,18 @@ def _create_stub_binary(
             actions = actions,
             apple_fragment = platform_prerequisites.apple_fragment,
             executable = "/usr/bin/xcrun",
-            arguments = ["bitcode_strip", "-r", "__BAZEL_XCODE_SDKROOT__/{}".format(xcode_stub_path), "-o", binary_artifact.path],
-            mnemonic = "BitcodeStripStub",
+            arguments = [
+                "lipo",
+                "__BAZEL_XCODE_SDKROOT__/{}".format(xcode_stub_path),
+                "-output",
+                binary_artifact.path,
+                "-extract",
+                platform_prerequisites.apple_fragment.single_arch_cpu,
```

Comment:
one concern i have here is will it always only be 1 arch these days? might not be true for watchOS now?